.github/workflows/publish.yml

```diff
@@ -53,16 +62,18 @@ jobs:
               uses: docker/metadata-action@v5
               env: ${{ fromJSON(steps.build_vars.outputs.vars) }}
               with:
-                  images: docker.io/aiidalab/${{ matrix.target }}
+                  # e.g. ghcr.io/aiidalab/full-stack
+                  images: ${{ inputs.registry }}/${{ github.repository_owner }}/${{ matrix.target }}
                   tags: |
+                      type=ref,event=pr
```

Comment:
If I understand correctly, the `release` workflow will triggered not only by the tag but also for every PR and this is why you include it here to have ghcr.io to have PR tag as we had before.

Reply:
Yes, I am reusing this workflow to publish images both to ghcr.io and docker.io. The former runs always, while the latter runs only on release or push to main. See the main.yml file.